app/_redirects

```diff
@@ -99,6 +99,7 @@
 /mesh/policies/meshgatewayroute /mesh/policies/meshhttproute/
 /mesh/policies/proxy-template  /mesh/policies/
 
+/kubernetes-ingress-controller/:version/installation/* /kubernetes-ingress-controller/install/
```

Comment:
I've checked older versions as well as latest, I can't see any /installation page that ever existed